dallinger/frontend/templates/base/dashboard.html

```diff
@@ -5,7 +5,7 @@
         <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
         <title>{% block title %}{{title}} - {{app_id}} Dashboard{% endblock %}</title>
         {% block replace_stylesheets %}
-          <link rel="stylesheet" href="/static/css/bootstrap.min.css" type="text/css">
+          <link rel="stylesheet" href="/static/css/dashboard-bootstrap.min.css" type="text/css">
```

Comment:
Why `dashboard-bootstrap.min.css` if login.html already references a (missing) `bootstrap.min.css`?

Reply:
I missed that. If it's called `bootstrap.min.css` the old version from the demos get copied instead. Having a different name allows us to use a different version from what a demo (or custom experiment) include for their own pages.

Reply:
Sorry, I meant `layout.html` not `login.html`. Isn't it better to just have a single copy of bootstrap that might be overwritten by a demo, rather than two (one of which is missing unless a demo installs it)?

Reply:
The problem is, we develop _using_ a demo (or a least I generally do) and I don't want the old bootstrap from the demo, I want the new one. I agree that it's dumb to depend on a demo or experiment to provide a CSS file, but that's also what we do now. I did what I did so we could move forward without needing to test demos. I'm also OK just loading bootstrap locally myself so I can design things for the day in the future when we officially upgrade bootstrap everywhere.

Reply:
Maybe we delete bootstrap in the demo that you are testing with, but leave the rest as is?

Reply:
OK, I'm on board with that, since it's a small step in the direction we want to move, and the weirdness might motivate us to keep moving. Bartlett's the lucky/unlucky one.